client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -45,6 +45,51 @@ export const getStateData = ( country, state ) => {
 	return stateData;
 };
 
+/**
+ * Returns an appropriate default currency (code) for
+ * the given country (code).
+ * @param {string} country Country (code) to get currency code for
+ * @return {string} best default currency code for country
+ */
+export const getCurrencyCodeForCountry = country => {
+	const countryData = getCountryData( country );
+	if ( ! countryData ) {
+		return 'USD';
+	}
+
+	return get( countryData, 'currency', 'USD' );
+};
+
+/**
+ * Returns an appropriate default dimension unit for
+ * the given country (code).
+ * @param {string} country Country (code) to get currency code for
```

Comment:
s/currency code/dimension unit/

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/21228/commits/9e47f657eb80a118bbbbb9d56f69413f719c705b